backend/dal/Migrations/v01.06.05/Up/PreUp/01-BuildingPredominateUses.sql

```diff
@@ -0,0 +1,20 @@
+DECLARE @correctPredominateUseId int;
+DECLARE @incorrectPredominateUseId int;
+
+select @correctPredominateUseId = Id from BuildingPredominateUses where Name = 'Senior Housing (Assisted Living / Skilled Nursing)';
+select @incorrectPredominateUseId = Id from BuildingPredominateUses where Name = 'Senior Housing (Assisted Living/Skilled Nursing)';
+update buildings set BuildingPredominateUseId = @correctPredominateUseId where BuildingPredominateUseId = @incorrectPredominateUseId;
+
+select @correctPredominateUseId = Id from BuildingPredominateUses where Name = 'Transportation (Airport / Rail / Bus station)';
+select @incorrectPredominateUseId = Id from BuildingPredominateUses where Name = 'Transportation (Airport/Rail/Bus station)';
+update buildings set BuildingPredominateUseId = @correctPredominateUseId where BuildingPredominateUseId = @incorrectPredominateUseId;
+
+select @correctPredominateUseId = Id from BuildingPredominateUses where Name = 'University/College';
+select @incorrectPredominateUseId = Id from BuildingPredominateUses where Name = 'University / Collect';
```

Comment:
The fix for the spacing around the / is in the next section.
Strange that most of the duplicates are so close but University/College does not look very similar to University / Collect